modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-sc-vsphere_{context}"]
+= vSphere automatic migration
+
+=== New installations of {product-title}
+For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
+
+=== Upgrading {product-title}
+When upgrading from {product-title} 4.12, or earlier, to 4.13, or later, automatic CSI migration for vSphere only occurs if you opt in.
```

Comment:
> to 4.13, or later, automatic CSI migration for vSphere only occurs if you opt in.

I think probably in 4.14, automatic vSphere CSI migration should be enabled by default, so I suggest remove `or later,` here.

Reply:
Fixed.